packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -77,14 +77,17 @@ function webpackConfigSingleSpa(opts) {
         },
       ],
     },
-    devtool: "sourcemap",
+    devtool: "source-map",
     devServer: {
       compress: true,
       historyApiFallback: true,
       headers: {
         "Access-Control-Allow-Origin": "*",
       },
-      disableHostCheck: true,
+      firewall: false,
+      client: {
+        port: 8080,
```

Comment:
`sockPort` was removed - this is the equivalent.

Reply:
It's much much better not to have to hard code the port here, but let the user specify it via CLI arg - we likely will need to fix webpack-dev-server in order for it to support this. It's one of the things holding up this PR